rollingpin/args.py

```diff
@@ -192,6 +187,27 @@ def _add_deploy_arguments(config, parser):
     )
 
 
+def _parse_command_args(command_args):
+    rv = []
+    for command in command_args:
+        (cmd_name, args) = (command[0], command[1:])
+
+        if cmd_name == "synchronize":
```

Comment:
not sure if there's a more pythonic way to transform the command string into types than a giant switch statement?

Reply:
the pythonic way is to use a dictionary.

Reply:
something like

```python
COMMANDS_BY_NAME = {
    "synchronize": commands.SynchronizeCommand,

}

...

def _parse_command_args(command_args):
    ...
    for command in command_args:
        ...

        command_cls = COMMANDS_BY_NAME[cmd_name]
        command_cls(args)
```